.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/qpp-conversion-tool
+    docker:
+      - image: circleci/openjdk:8-jdk-browsers
+    steps:
+      - checkout
+      - run: cd converter; mvn dependency:resolve
+      - run: wget -O sonarscanner.zip https://sonarsource.bintray.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-3.0.1.733.zip
+      - run: unzip -d . sonarscanner.zip
+      - run: wget -O get-pip.py https://bootstrap.pypa.io/get-pip.py
+      - run: sudo python get-pip.py
```

Comment:
There's no way to specify an environment that has Python and PIP already installed in CircleCI 2.0?  We need to do this manually?

Reply:
I still had to install pip when I used the python container as well. It had both python 2 and 3 installed, but for whatever reason neither came with pip.